pages/index.js

```diff
@@ -1,21 +1,52 @@
-import SEO from '@/components/Seo'
+import { useEffect } from 'react'
+import { prefetch } from '@layer0/prefetch/window/prefetch'
 
-const Home = () => {
-  const meta = {
-    title: 'Layer0 Nextjs Example',
-    description: 'This open source project demonstrates Prefetching, and Image Optimization with Layer0 using Nextjs.',
-    url: 'https://layer0-docs-layer0-next-example-default.layer0.link',
-    image: 'https://layer0-docs-og-image-default.layer0.link/api?title=Layer0 Nextjs Example&width=1400&height=720'
-  }
+const Home = ({}) => {
+  // Prefetch grey image using Layer0 for instant placeholder loads on the PLP page
+  useEffect(() => {
+    prefetch('/images/grey.png')
```

Comment:
Not sure we need this image, i know its for the other pages, but see the comments there.